src/mongo/MongoScrapbook.ts

```diff
@@ -372,23 +374,24 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 			} catch (error) {
 				let err: IParsedError = parseError(error);
 				this.addErrorToCommand(err, ctx);
+				return {};
 			}
 		}
-		return constructedObject;
+		return { $oid: constructedObject.toString() };
```

Comment:
If you hit the error case, this will cause null ref exception

Reply:
Do you have a testcase for that scenario?

Reply:
The error catch block returns an empty object, and [this](https://github.com/Microsoft/vscode-cosmosdb/blob/a0fafcab0093a7ac23f2bcca40d3aadf809b323d/test/mongoGetCommand.test.ts#L691) test case handles an error case. Can you give me more detail on what example you're expecting?

Reply:
My fault, could've sworn there was no return in the error case.